Bylaws.md

```diff
@@ -196,7 +204,7 @@ Members have discretion about which mailing list they use, but are strongly enco
 
 Members are strongly discouraged from posting the text of Member Mail List messages to the Forum Public Mail List without the permission of the author or commenter.
```

Comment:
This should not only talk about the Forum Public Mail List:

```suggestion
Members are strongly discouraged from posting the text of Member Mail List messages to a Public Mail List without the permission of the author or commenter.
```

Reply:
Sounds like a good improvement. Leaving it open for more possible comments.